tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,27 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+from gaiatest.apps.bzlite.app import BugzillaLite
+
+
+class TestFillBug(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.test_data = {
+            'name': 'Bugzilla Lite',
+            'url': 'http://bzlite-staging.herokuapp.com/manifest.webapp'}
+        self.logger.info('Test data: %s' % self.test_data)
```

Comment:
Why does this need to be shown in the console? I mean, the test itself is already showing what self.test_data is.